tests/main/security-apparmor/task.yaml

```diff
@@ -0,0 +1,22 @@
+summary: Check basic apparmor confinement rules.
+
+prepare: |
+    echo "Given a basic snap is installed"
+    snapbuild $TESTSLIB/snaps/test-snapd-tools .
+    snap install test-snapd-tools_1.0_all.snap
+
+restore: |
+    rm -f test-snapd-tools_1.0_all.snap
```

Comment:
@zyga thankfully we have a common restore step that gets rid of all the state acquired (including installed snaps) during the tasks :)

Cheers!